ui/packages/consul-ui/app/abilities/partition.js

```diff
@@ -0,0 +1,25 @@
+import BaseAbility from './base';
+import { inject as service } from '@ember/service';
+
+export default class PartitionAbility extends BaseAbility {
+  @service('env') env;
+
+  resource = 'operator';
+  segmented = false;
+
+  get canManage() {
+    return this.canCreate;
+  }
+
+  get canDelete() {
+    return this.item.Name !== 'default' && super.canDelete;
+  }
+
+  get canChoose() {
+    return this.canUse && (this.partitions || []).length > 0;
+  }
+
+  get canUse() {
+    return this.env.var('CONSUL_PARTITIONS_ENABLED');
+  }
+}
```

Comment:
This is just copypasta from the NsapceAbility for now. I need to double check that it requires the same `operator` resource.

Reply:
From speaking with others on the team sounds like there is a `mesh` resource to check to see whether I can actually 'manage' (i.e. CRUD) partitions. I'll confirm this with the rest of the team and do this as part of the application level changes (rather than the HTTP/model layer changes here)